dev-client/src/components/content/typography/TranslatedParagraph.tsx

```diff
@@ -15,32 +15,24 @@
  * along with this program. If not, see https://www.gnu.org/licenses/.
  */
 
-import {Trans} from 'react-i18next';
-
+import {
+  TranslatedContent,
+  TranslatedContentProps,
+} from 'terraso-mobile-client/components/content/typography/TranslatedContent';
 import {Text} from 'terraso-mobile-client/components/NativeBaseAdapters';
 
-type TranslatedBodyProps = {
-  i18nKey: string;
+type TranslatedParagraphProps = {
   variant?: 'body1' | 'body2';
-  values?: {};
-};
+} & TranslatedContentProps;
 
-export function TranslatedBody({
-  i18nKey,
+export function TranslatedParagraph({
```

Comment:
This and similar components were renamed to match design elements in the typography section of the component library.